CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## 1.10.1-dev
+* Added the option for custom `zoneValues` on `Chain.capture` 
```

Comment:
Please document the exact param name. Also the version should be changed to `1.11.0-dev` since we added a new feature

Reply:
bumped the version and updated the changelog. Dunno if the entry is good thought, english is not my main language